app/controllers/taxons_controller.rb

```diff
@@ -40,13 +40,24 @@ def create
 
   def show
     respond_to do |format|
+      email_lists = []
+      begin
+        email_lists = Services.email_alert_api.find_subscriber_list(links: { taxon_tree: [taxon.content_id] })
+      rescue GdsApi::BaseError => e
+        GovukError.notify(e)
+      end
+      email_list = email_lists[0] || {}
+
       format.html do
         render locals: {
-          page: Taxonomy::ShowPage.new(taxon, params.fetch(:viz, 'taxonomy_tree'))
+          page: Taxonomy::ShowPage.new(taxon, params.fetch(:viz, 'taxonomy_tree'), email_list)
```

Comment:
It might be cleaner to have the email list stuff in the ShowPage class, rather than the controller. Similar to how tagged is handled here https://github.com/alphagov/content-tagger/blob/7e6177d912e4e33ee7033d979667c2ea31000ce0/app/services/taxonomy/show_page.rb#L47-L57

Reply:
I put it in the controller so the json could include the number of subscribers, but if that's not necessary I'll move it.

Reply:
I'm not quite sure if anything is using the JSON format, so I'd be tempted to ignore that.